### ISSUE TITLE
Server crashing (Ticking entity error) when extracting liquids with cells from tree fluid extractor

### ISSUE THREAD

**Issue by DerSoftwerker** [FTBTeam/FTB-Continuum#87]
* **Modpack Version**: 1.0.2

* **Issue**: Server crashing (Ticking entity error) when extracting liquids with cells from tree fluid extractor

* **Link to Log or Crash File Paste**: https://paste.feed-the-beast.com/view/59c0770f

* **Is it Repeatable?**: Not tried, live environment. When issue occurs, the server will be in a crash loop

* **Mod/s Affected**: Tech reborn

* **Known Fix**: Forge.cfg - remove erroring entities true

**Comment by Valerate** [comment]
Can confirm, Server is now stuck on crash loop.
The error I got when the server died was "[java.lang.RuntimeException: Cannot Hotload Dim: Overworld is not Loaded!](https://pastebin.com/yhHqhuPZ)" I stood next to the machine so I doubt it wasn't loaded. .
My Added mods; Millenaire, Minecolonies, neither which I've used much, and Infernal Monsters.
At the time of the crash I was removing a cell of Latex from the machine and the other person on the server was exploring.

Addition: I managed to not crash it once, but it consumed everything inside (8000mb) and only gave me 1 cell
